packages/hot-updater/src/utils/version/getAndroidVersion.ts

```diff
@@ -0,0 +1,46 @@
+import path from "path";
+import { getCwd } from "@hot-updater/plugin-core";
+import fs from "fs/promises";
+import semverValid from "semver/ranges/valid";
+
+const getAndroidVersionFromAppBuildGradle = async (): Promise<
+  string | null
+> => {
+  const buildGradlePath = path.join(getCwd(), "android", "app", "build.gradle");
+
+  try {
+    const buildGradleContent = await fs.readFile(buildGradlePath, "utf8");
+    const versionNameMatch = buildGradleContent.match(
+      /versionName\s+['"]([^"]+)['"]/,
+    );
+    return versionNameMatch?.[1] ?? null;
+  } catch (error) {
+    return null;
+  }
+};
+
+const Strategy = {
```

Comment:
```suggestion
const AndroidVersionStrategy = {
```
?

Reply:
I see, the naming of `Strategy` feels exaggeration. I have changed!